frontend/src/redux/modules/reducer.js

```diff
@@ -1,13 +1,11 @@
 import { connectRouter } from 'connected-react-router'
 import { combineReducers } from 'redux'
 import user from './user'
-import auth from './auth'
 
 const createRootReducer = history =>
   combineReducers({
     router: connectRouter(history),
-    user,
-    auth

```

Comment:
loading was the only thing that was used in auth and it isn't really auth, so I just moved it to user